### PR TITLE
Add more details in the README.md

### PR DESCRIPTION
**Pull request by doxiao**
Add details about how to mount host dir to a container's file system when a customer provides his/her own SSL config.
Signed-off-by: doxiao <dongbo.xiao@oracle.com>

@mriccell @rosemarymarano

### CODE REVIEW

File: OracleWebLogic/samples/12213-webtier-apache/README.md [hunk 1]
```diff
@@ -122,15 +122,19 @@ In production, Oracle strongly recommends that you provide your own certificates
...
                      -e WEBLOGIC_HOST=<admin_host> \
                      -e WEBLOGIC_PORT=7001 \
                      -p 4433:4433 \
+                     -v <host-config-dir>:/config \
+                     -w /config \
                      oracle/apache:12.2.1.3
 
 Use `SSL_CERT_FILE` and `SSL_CERT_KEY_FILE` to specify the name of the certificate and key files, including the path in the container's file system. Both of the environment variables need to be set.
 
+The example above assumes that your SSL certificate and key files are located in `<host-config-dir>`. It demonstrates how to mount the files into `/config` dir in the container's file system. You can mount `<host-config-dir>` into any directory in the container's file system, for example, `/myconfig`.
```
Comment (rosemarymarano) [inline OracleWebLogic/samples/12213-webtier-apache/README.md]: "to mount the files into "the" `/config` dir in the container's file system.

**Comment by doxiao** [comment]
Monica, somehow I accidentally closed this request. Feel free to reopen it if you have not done with it. Thanks.

**Comment by mriccell** [comment]
I can not reopen maybe you can.  I need to merge the changes and I did not have a chance to do so.

**Comment by Djelibeybi** [comment]
This cannot be re-opened. You will need to submit a new pull request.

**Comment by doxiao** [comment]
I was trying to merge and resolve the conflicts, and as a result, I accidentally closed the request. Let me recreate one.

**Pull request closed without merging**